src/cargo/ops/cargo_package.rs

```diff
@@ -786,3 +767,28 @@ fn check_filename(file: &Path) -> CargoResult<()> {
     }
     Ok(())
 }
+
+lazy_static::lazy_static! {
```

Comment:
I think it's fine to avoid a `lazy_static` here, it's fine if the times differe by a bit I just wanted to avoid the dance to get the seconds out from being repeated 3 times (which using a helper function fixes)

Reply:
The `lazy_static` is just the last commit. Would you prefer a revert commit or just drop the last commit?

Reply:
It's fine to force-push the branch to drop the commit